hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -3186,6 +3201,7 @@ bool MDMParser::smsRead(int ix, char* num, char* buf, int len)
     param.num = num;
     param.buf = buf;
     sendFormated("AT+CMGR=%d\r\n",ix);
+    // FIXME: check for SMS buffer size when smsRead() is being used
```

Comment:
This function `smsRead()` is currently not being used. And also unclear that this needs a fix?

Reply:
Probably ok to leave it for now since it's not an official API